src/main/groovy/com/netgrif/application/engine/petrinet/domain/dataset/logic/action/ActionDelegate.groovy

```diff
@@ -458,59 +474,89 @@ class ActionDelegate {
         }]
     }
 
-    protected SetDataEventOutcome createSetDataEventOutcome() {
-        return new SetDataEventOutcome(this.useCase, this.task.orElse(null))
+    def makeInterProcess(List<String> fieldIds, Closure behavior) {
```

Comment:
Is it necessary to have new function name? Cannot be overloaded existing `make` function?

Reply:
``make`` has an input argument of type ``List`` and ``Closure`` and this function has the same types. If they have the same name, Java/Groovy is not able to decide, which one to use.